test/test_helper.rb

```diff
@@ -3,11 +3,25 @@
 require 'rails/test_help'
 require 'capybara/rails'
 require 'minitest/emoji'
+require 'database_cleaner'
```

Comment:
I wish we could turn off hound on certain things like this. You stayed consistent with single quotes because that's what was there in the first place.